### ISSUE TITLE
[Feature Request] Warning for bollards on a cycling route 

### ISSUE THREAD

**Issue by pietervdvn** [osmandapp/Osmand#4996]
Hi,

It would be usefull that OsmAnd would announce a "Warning: obstacle" if a `barrier=bollard` is on a cycleway. This could prevent accidents, as people are remembered to look out for the bollard.

**Comment by lokaman** [comment]
It's probably more useful to look on the road ahead, instead of looking at the display wich tells you to look at the road?!

**Comment by pietervdvn** [comment]
Hey lokaman,

I don't see your comment anymore; but I do mean the spoken instructions
which would warn, similar to the warnings for pedestrian crossings and
speed bumps.

You are definitely right that a warning on the screen is pretty useless.


Met vriendelijke groeten,
Pieter Vander Vennet

2018-02-13 11:00 GMT+01:00 lokaman <notifications@github.com>:

**Comment by lokaman** [comment]
Yeah, i realized i missunderstood your request. I had nothing left to say in this case.
Personally i just look at the street and see the obstacles. And if im not sure if there is one, i cycle slower.

I just dont understand this "we need tech to tell us to be responsible".
Its not that hard to be drive / cycle in a responsible manner.
And as mapping is incomplete regarding bollards / barriers the best advise still remain:

"Look on the street, be responsible for your actions".

Kind regards :)

**Comment by pietervdvn** [comment]
Ironically, I recently hit a bollard because I was distracted (making notes
for mapping).

It would have helped then :p

And this is mainly a feature that would be usefull when cycling in group:
sometimes the last cyclists can't see the obstacles; a warning then is
usefull.


Met vriendelijke groeten,
Pieter Vander Vennet

2018-02-14 12:30 GMT+01:00 lokaman <notifications@github.com>:

**Comment by lokaman** [comment]
Attention to the road! :wink:

However I still don't think we need this feature. But we will see if there's somebody to share his / her opinion onto the topic.

Wish you a safe ride! :relaxed:
